### PR TITLE
Add to index.html and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by YashJain1999**
Here is my contribution to your repo
Thank me later

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,11 +4,14 @@
...
     <title>HTML5 Test Page</title>
+    <link rel="stylesheet" href=".git/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu&display=swap" rel="stylesheet">
   </head>
   <body>
+    <marquee>HTML-5</marquee>
     <div id="top" class="page" role="document">
       <header role="banner">
-        <h1>HTML5 Test Page</h1>
+        <h1 id="head">HTML5 Test Page</h1>
```
Comment (gilluminate) [inline index.html:14]: why is this ID necessary?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,3 +5,8 @@
 - Pull requests must be accompanied by passing all existing automated tests (`$ npm test`).
 
 Thanks in advance for helping out!
+
+I have tried to add some more things into your repo like -
+
+1. Marquee tag in html5
+2. Style the page with the help of CSS like styling the image, headings and background-color etc.
```
Comment (gilluminate) [inline CONTRIBUTING.md:12]: I believe adding styles defeats the purpose of the page

Comment (gilluminate) [inline CONTRIBUTING.md:12]: This message does not belong in the Contributing document, leave this to the PR message.

**Comment by cbracco** [comment]
Closing this one out.

**Pull request closed without merging**